.gitignore

```diff
@@ -109,3 +109,6 @@ Icon
 Network Trash Folder
 Temporary Items
 .apdisk
+
+# VirtualEnv
+hknweb_dev
```

Comment:
usually I call this folder `venv` or `.venv`, `hknweb_dev` is pretty similar to the source folder `hknweb`.

Reply:
`.venv` it is!